### DISCUSSION TITLE
Search by city on data tracker

### DISCUSSION THREAD

**Discussion by benhammondmusic** [SatcherInstitute/health-equity-tracker#4291] [HET Ideas (Features, Requests, New Data, etc)]
![search by city](https://user-images.githubusercontent.com/41567007/131157741-bb7584fd-edc0-40ab-a935-dce3c7a9b72b.png)

Currently the tracker location search box is unable to handle city requests; this is suboptimal as many people might a) attempt a search by city as their first search and b) not even know the name of the county they live in or especially not know the names of their surrounding counties. 

Proposal to implement a lookup and if there is a matching city (or cities) to display those as options with the containing county, which will then map on click to the corresponding county

**Comment by jgonzalezmsm** [comment]
This is a really cool idea and nice workaround to not having cities in the data

**Comment by jgonzalezmsm** [comment]
add the word "county" after every county name on region select.

**Comment by jgonzalezmsm** [comment]
can search not be an exact match?

**Comment by benhammondmusic** [comment]
mui autocomplete allows different variables for the `value` and the `displayValue`; this could be one implementation, where city names are entries that contain the `displayValue` of the city but the `value` of the parent county (since county level is the lowest level we can show on the tracker)

**Comment by benhammondmusic** [comment]
> add the word "county" after every county name on region select.

Addressed in #1310

**Comment by benhammondmusic** [comment]
> can search not be an exact match?

Filed as new issue #1311

**Comment by benhammondmusic** [comment]
Promoted to issue #4918 in the new [Better Location Search](https://github.com/SatcherInstitute/health-equity-tracker/milestone/56) milestone, alongside fuzzy matching (#4917).
